cost-analyzer/templates/grafana-dashboard-cluster-metrics-template.yaml

```diff
@@ -1,3 +1,7 @@
+{{- if .Values.grafana -}}
+{{- if .Values.grafana.sidecar -}}
+{{- if .Values.grafana.dashboards -}}
+{{- if .Values.grafana.dashboards.enabled -}}
```

Comment:
This is undesirable overkill
```
{{- if .Values.grafana -}}
{{- if .Values.grafana.sidecar -}}
{{- if .Values.grafana.dashboards -}}
{{- if .Values.grafana.dashboards.enabled -}}
```
All we need or want for the dashboards is `{{- if .Values.grafana.dashboards.enabled -}}`

If you want to get fancy, you can also add a value specific to each dashboard, such as `{{- if .Values.grafana.dashboards.clusterMetrics.enabled -}}`

Reply:
These conditionals are defensive programming to protect people using --reuse-values from earlier values.yaml, which we in general support in version 1.X.X. Supplying a default isn't enough We can look into simplifying in a later version.

Reply:
@AjayTripathy okay